static/packages/autocomplete.js

```diff
@@ -59,26 +59,22 @@ class Address {
         return address;
     }
 
-    extractStreetDetails(self, detailsStr) {
-        let word = "";
+    extractStreetDetails(self, streetDetails) {

```

Comment:
Why not use regex?

Reply:
Yeah, they are cool, but I didn't go too far since there are a lot of theme. Did a very basic thing but it still helped quite a bit.